### ISSUE TITLE
Change 3.3.1: Error Suggestion to 3.3.3: Error Suggestion, 3.3.1: Error Identification

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#1797]
https://www.w3.org/WAI/WCAG22/Understanding/error-identification

Last sentence before the Benefit says:

> See also 3.3.1: Error Suggestion.

This of course should be:

> See also 3.3.3: Error Suggestion.

**Comment by patrickhlauke** [comment]
and probably just above that, where it says "Currently, few technologies support..." - that can probably now be removed, as programmatic identification of errors etc is fairly well supported nowadays (in native HTML, as well as in combination with ARIA)
